global.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
but the hack to make `dotnet` behave is simple enough too

![](http://i.imgur.com/sgfltoj.png)